src/module-3/online-store/styles/base/_card.scss

```diff
@@ -0,0 +1,115 @@
+@use 'global' as *;
+
+.card {
+  height: 486rem;
+  //max-width: 400rem;
+  width: 100%;
+  @extend %def-box;
```

Comment:
always put `@extend` stuff on top; otherwise it may override the styles you wrote earlier.  
Avoid overriding extended styles on purpose.